README.md

```diff
@@ -1 +1,212 @@
-# Template
\ No newline at end of file
+# Ansible Playbook to deploy Solana Node
```

Comment:
Please refer this for inspiration of our readme https://www.notion.so/encapsulate/Solana-21d14f565a508089adefe2dfb41934d6?source=copy_link#23014f565a50803e97bdf5f33a6a7b48